Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -293,4 +293,14 @@ final class TestCommandTests: CommandsTestCase {
             }
         }
     }
+
+#if !canImport(Darwin)
+    func testGeneratedMainIsConcurrencySafe_XCTest() throws {
+        let strictConcurrencyFlags = ["-Xswiftc", "-strict-concurrency=complete"]
+        try fixture(name: "Miscellaneous/TestDiscovery/Simple") { fixturePath in
+            let (_, stderr) = try SwiftPM.Test.execute(strictConcurrencyFlags, packagePath: fixturePath)
+            XCTAssertNoMatch(stderr, .contains("is not concurrency-safe"))
```

Comment:
Might it make sense to broaden this check to look for any warnings at all in the build log?

Reply:
That might be worthwhile as a separate test: this test is specifically confirming that the concurrency issue is resolved.